config/config_test.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"testing"
+
+	"sigs.k8s.io/yaml"
+)
+
+func TestBasicMarshalling(t *testing.T) {
+	var cfg Config
+
+	cfg.Gate.Endpoint = "test"
+
+	bs, err := yaml.Marshal(cfg)
```

Comment:
Maybe this is a go convention?  Perhaps a more descriptive variable name than `bs`?

Reply:
In this specific case, it's short for "byte slice". Being a local variable and all, I don't mind what it's called, changing to `byteSlice`.

Reply:
sorry, my mind went straight to that other thing bs sometimes stands for :)